src/main/java/dk/kb/datahandler/oai/OaiHarvestClient.java

```diff
@@ -84,9 +88,9 @@ public OaiResponse next() throws Exception{
         String  resumptionToken=  getResumptionToken(document);
         oaiResponse.setTotalRecords(getResumptionTotalSize(document));        
 
-        if (resumptionToken != null && !resumptionToken.equals("")) {
+        if (resumptionToken != null && !resumptionToken.isEmpty()) {
             this.resumptionToken = resumptionToken;  
-      log.info("next resumption token:"+resumptionToken);
+            log.info("next resumption token:"+resumptionToken);
```

Comment:
Not your code, but could you lower this to `debug`? Seems much too spammy for my taste and the resumption token is logged indirectly at line 71.